exist-core/src/main/antlr/org/exist/xquery/parser/XQuery.g

```diff
@@ -2224,6 +2224,12 @@ reservedKeywords returns [String name]
 	"map" { name = "map"; }
 	|
 	"array" { name = "array"; }
+	|
+	"copy-namespaces" { name = "copy-namespaces"; }
+	|
+    "empty-sequence" { name = "empty-sequence"; }
```

Comment:
It appears as if the indentation is not correct.

Reply:
@line-o Thanks! I just force pushed an update that fixes that.